SU2_CFD/src/drivers/CMultizoneDriver.cpp

```diff
@@ -139,6 +139,27 @@ CMultizoneDriver::~CMultizoneDriver(void) {
 
 void CMultizoneDriver::StartSolver() {
 
+  /*--- Find out the minimum of all references times and then set each zone to this (same) value.
+   * (To ensure that all zones run synchronously in time, be it a dimensional or non-dimensionalized one.) ---*/
+
+  su2double Time_Ref = config_container[ZONE_0]->GetTime_Ref();
+
+  for (iZone = 1; iZone < nZone; iZone++) {
+    if ((config_container[iZone]->GetTime_Ref() < Time_Ref)
+        && (config_container[iZone]->GetKind_Solver() != FEM_ELASTICITY))
+      Time_Ref = config_container[iZone]->GetTime_Ref();
+  }
```

Comment:
Update: This is a lot better as is does not rely on a hard-coded value like ZONE_0 but still..
Anyway, somehow we need to set the same reference time in all zones before starting an unsteady run.